src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp

```diff
@@ -425,6 +438,11 @@ class CoupledSolver : public SolverBase
 
       if( isConverged )
       {
+        // Save Time step statistics for the subsolvers
+        forEachArgInTuple( m_solvers, [&]( auto & solver, auto )
+        {
+          solver->getSolverStatistics().saveTimeStepStatistics();
```

Comment:
This will most likely require a rebaseline. Can you double check?

Reply:
This only affects screen output, should not cause any diffs in integratedTests

Reply:
You may be modifying a member
```
  registerGroup( groupKeyStruct::solverStatisticsString(), &m_solverStatistics );
```
that may be written as well (and compared). But I confess I really don't know.

@CusiniM Should we proceed? I'm next in line so I'm fine fixing it if it appears something's needed.

Reply:
I really think that it will require a rebaseline. It's fine if you want to do a single one for both PRs though.

Reply:
OK then. How should we proceed?

Reply:
well, if you don't mind dealing with the diffs introduced by this PR when you rebase for yours I think you can merge this one and then go ahead and work on the rebaseline.